lib/tests/test_mutations.c

```diff
@@ -532,11 +532,8 @@ test_single_tree_mutgen_keep_sites_many_mutations(void)
     ret = mutgen_set_rate(&mutgen, 10);
     CU_ASSERT_EQUAL_FATAL(ret, 0);
 
-    for (j = 0; j < 10; j++) {
-        ret = mutgen_generate(&mutgen, MSP_KEEP_SITES);
-        CU_ASSERT_EQUAL_FATAL(ret, 0);
-    }
-    CU_ASSERT_TRUE(tables.sites.num_rows > 2);
+    ret = mutgen_generate(&mutgen, MSP_KEEP_SITES);
+    CU_ASSERT_EQUAL_FATAL(ret, MSP_ERR_BAD_ANCESTRAL_MUTATION);
```

Comment:
seems like we should still be testing the ability to add mutations - we could do this by adding new mutations with a different alphabet

Reply:
We're testing it in other places all right. I tried using a different alphabet here, but we get MSP_ERR_UNKNOWN_ALLELE.